docs.kosli.com/content/tutorials/unauthorized_iac_changes.md

```diff
@@ -0,0 +1,166 @@
+To detect when an **unauthorized** change happens, we can monitor the state file for changes and record those changes in
+a Kosli environment.
+
+Let's start by creating an environment of type `server`. 
+
+```shell {.command}
+$ kosli create env terraform-state --type server
+```
+
+We can report the state file to the environment we created:
+
+{{<hint info>}}
+In this tutorial, we run the environment reporting manually. 
+In production, you would configure the environment reporting to run periodically or on changes.
+{{</hint>}}
+
+```shell {.command}
+$ kosli snapshot server terraform-state -p terraform.tfstate
+```
+
+You can get the latest snapshot of the environment by running:
+
+```shell
+$ kosli get snapshot terraform-state
+COMMIT   ARTIFACT                                                                        FLOW     RUNNING_SINCE   REPLICAS
+6cbdb34  Name: /Users/samialajrami/workspace/kosli/iac-changes-tutorial/terraform.tfstate  tf-tutorial  28 minutes ago  1
+         Fingerprint: a57667a7b921b91d438631afa1a1fe35300b4da909a19d2b61196580f30f1d0c 
+```
+
+Note that the `FLOW` column indicates that this artifact came from the `tf-tutorial` flow which means Kosli has provenance for 
+where this change came from.
+
+You can also view the environment status in the Kosli UI by navigating to: `Environments > terraform-state`.
+At this point you should see one artifact with a compliant status since we have provenance for the change that happened.
+
+{{< figure src="/images/tutorials/iac-changes/authorized-iac-change.png" alt="Environment shows an authorized change" width="90%" >}}
+
+## Introducing an unauthorized change
+
+Now let's see how Kosli can help catching an unauthorized change. 
+We can simulate such change by modifying the `random_pet_result` output on line 6 in main.tf to `random_pet_name` and running:
+
+```shell {.command}
+$ terraform apply --auto-approve
+```
+
+This updates the state file. Let's report the updated state file to the Kosli environment.
+
+{{<hint info>}}
+In production, this step won't be necessary because you would have configured environment reporting to happen
+automatically (either on state file change or periodically).
+{{</hint>}}
+
+```shell {.command}
+$ kosli snapshot server terraform-state -p terraform.tfstate
+```
+
+Getting the latest snapshot of the environment by running the command below shows that the `FLOW` is unknown. 
+This means that Kosli does not have provenance for that change (i.e. it is an unauthorized change).
+
+```shell
+$ kosli get snapshot terraform-state
+COMMIT  ARTIFACT                                                                        FLOW  RUNNING_SINCE  REPLICAS
+N/A     Name: /Users/samialajrami/workspace/kosli/iac-changes-tutorial/terraform.tfstate  N/A   8 minutes ago  1
+        Fingerprint: edd93dcde27718ed493222ceb218275655555f3f3bfefa95628c599e678ac325 
+```
+
+When you navigate to the environment page again, you will see a non-compliant artifact running.
```

Comment:
Interesting that we use the word `running` in the Environment UX.
A tf-state-file does not feel like it is "running"?